vignettes/articles/ExternalAPI.Rmd

```diff
@@ -1,4 +1,7 @@
-# Hector External API
+---
+title: "Hector External API"
+subtitle: "C++"
+---
 
```

Comment:
@bpbond is this correct & up todate?

Reply:
Just checking — I should go look at the entire Rmd file right?

Reply:
Yes, please!